docs/reference/eql-functions.md

```
@@ -9,7 +9,7 @@ A reference for the functions and operators EQL exposes for querying encrypted d
 - [Operators](#operators)
 - [Function Equivalents](#function-equivalents)
 - [Index Term Extraction](#index-term-extraction)
-- [Encrypted JSON (`public.eql_v3_json`)](#encrypted-json-eql_v3json)
+- [Encrypted JSON (`public.eql_v3_json_search`)](#encrypted-json-eql_v3json)
```

markdownlint-cli2 (0.23.0)

[warning] Link fragments should be valid

(MD051, link-fragments)